apps/web/src/design-system/tabs/Tabs.tsx

```diff
@@ -47,23 +47,30 @@ export const Tabs = React.forwardRef<HTMLDivElement, ITabsProp>(
         <MantineTabs
           orientation={orientation}
           ref={ref}
-          active={active}
+          value={value}
+          defaultValue={defaultValue}
           onTabChange={onTabChange}
-          position={position}
-          variant="unstyled"
+          variant="default"
           classNames={classes}
         >
-          {menuTabs.map((menuTab, i) =>
-            withIcon ? (
-              <MantineTabs.Tab label={menuTab.label} icon={menuTab.icon} key={i}>
-                {menuTab.content}
-              </MantineTabs.Tab>
-            ) : (
-              <MantineTabs.Tab label={menuTab.label} key={i}>
-                {menuTab.content}
-              </MantineTabs.Tab>
-            )
-          )}
+          <MantineTabs.List>
```

Comment:
the `Tabs` from the Mantine lib now requires to use `Tabs.List` to render the items and `Tabs.Panel` with `value` for the tab content